src/oaklib/cli.py

```diff
@@ -2286,23 +2287,65 @@ def descendants(
 @main.command()
 @click.argument("terms", nargs=-1)
 @click.option("-o", "--output", help="Path to output file")
+@output_type_option
 @click.option(
-    "--include-all-predicates/--no-include-all-predicates",
-    default=False,
-    show_default=True,
-    help="For formats that export only IS_A by default, this will include all possible predicates",
+    "-c",
+    "--config-file",
```

Comment:
**Docs**
I also documented this in `dump()` within `cli.py` So that there's an easy way to know what to do if you're interrogating the `--help` in the command line. But if this is too much, feel free to delete this text, and we can simply direct / link to `docs/howtos/fhir-conversions.rst` or refer them to the docstring for `OboGraphToFHIRConverter` (which doesn't say anything about a config file, but does show params).